### PR TITLE
(multiple) small ui fixes for the options ui

### PR DESCRIPTION
**Pull request by Duke-M-commits**
* fixed some cut off texts (english and german only)
image

* added some dots to sentences to improve reading consistency

### CODE REVIEW

File: resource/reactivedrop_english.txt [hunk 1]
```diff
@@ -5125,7 +5125,7 @@
 "rd_option_speed_auto_restart_mission"	"Automatic Mission Retry"
 "rd_option_speed_auto_restart_mission_hint"	"Return to briefing automatically after a mission is failed."
 "rd_option_leaderboard_private_stats"	"Hide My Stats"
-"rd_option_leaderboard_private_stats_hint"	"Removes most of your lifetime mission stats from your public commander profile. Your experience level and promotion, as well as any equipped medals are still visible, and your entries are still visible on individual leaderboards."
+"rd_option_leaderboard_private_stats_hint"	"Hides most lifetime stats from your public commander profile. Experience, promotion and medals are still visible, excluding entries on individual leaderboards."
```
Comment (BenLubar) [inline resource/reactivedrop_english.txt:5128]: I would interpret "excluding entries on individual leaderboards" to mean the opposite of the old text.

**Review by BenLubar — CHANGES_REQUESTED** [review]

File: resource/reactivedrop_english.txt (hunk 1 above)
Comment (BenLubar) [reply resource/reactivedrop_english.txt:5128]: maybe "as are" instead of "excluding"?

**Comment by Duke-M-commits** [comment]
Should I outdent the few translations been made which are obviously too long also?

**Comment by BenLubar** [comment]
> Should I outdent the few translations been made which are obviously too long also?

For now, just tag the translators of those specific lines and tell them to take a look.

**Comment by Duke-M-commits** [comment]
@Blueberryy
image

@MakinDay
image

**Pull request merged**